### ISSUE TITLE
Realsense D435 did not turn off after stopping the launch on Jetson Xavier

### ISSUE THREAD

**Issue by amburkoff** [realsenseai/realsense-ros#1236]
## Environment
|                           |                                                             |
|---------------------------|-------------------------------------------------------------|
| Computer                  | NVIDIA Jetson AGX Xavier Jetpack 4.3 [L4T 32.3.1]           |
| Jetson_clocks             | On                                                          |
| Autosuspend               | -1                                                          |
| OS                        | Ubuntu 18.04                                                |
| ROS verison               | Melodic                                                     |
| Realsense type            | D435                                                        |
| Camera firmware           | 05.12.05                                                    |
| Realsense connection type | type-c to type-c cable (TetherPro 1.8m) that Intel recommended |
| LibRealSense version      |  v2.33.1 (install through apt)                              |
| RealSense ROS version     |  v2.2.13 (install through apt)                              |

We conducted camera fault tolerance tests. The first test was long time test(14 ours). During this time camera worked fine. 
After that, we immediately conducted short tests (one minute each). After the fifth test, the camera did not turn off after stopping the launch through "Ctrl-C". (IR projector continued to light). Then we tried to conduct another short test, but the realsense node could not connect to the camera, with the following error:
```
Wed 2020-06-10 09:32:48.831: realsense2_camera_manager:  [33m[RealSenseNodeFactory::getDevice]: Device 1/1 failed with exception: set_xu(ctrl=1) failed! Last Error: Resource temporarily unavailable [0m
Wed 2020-06-10 09:32:48.831: realsense2_camera_manager:  [31m[RealSenseNodeFactory::getDevice]: The requested device with serial number 834412070084 and usb port id 4-1 is NOT found. Will Try again. [0m
```
And tried to reconnect the camera in a loop.

The launch command is:
```
mon launch realsense2_camera rs_camera.launch serial_no:=834412070084 usb_port_id:=4-1 --log="/home/courier/realsense_tests/test_28"
```
We use mon launch just to easily safe logs.

## Log files:
- test_27.txt - last fine short test (after this one, camera didn't turn off)
- test_28.txt - camera didn't connect
- test_29.txt - camera didn't connect
- list_27.txt - list of usb devices before execute launch
- list_27_2.txt - list of usb devices after execute launch
- list_28.txt - list of usb devices before execute launch
- list_28_2.txt - list of usb devices after execute launch
- list_29.txt - list of usb devices before execute launch
- list_29_2.txt - list of usb devices after execute launch

[list_27.txt](https://github.com/IntelRealSense/realsense-ros/files/4758732/list_27.txt)
[list_27_2.txt](https://github.com/IntelRealSense/realsense-ros/files/4758734/list_27_2.txt)
[list_28.txt](https://github.com/IntelRealSense/realsense-ros/files/4758736/list_28.txt)
[list_28_2.txt](https://github.com/IntelRealSense/realsense-ros/files/4758737/list_28_2.txt)
[list_29.txt](https://github.com/IntelRealSense/realsense-ros/files/4758738/list_29.txt)
[list_29_2.txt](https://github.com/IntelRealSense/realsense-ros/files/4758739/list_29_2.txt)
[test_27.txt](https://github.com/IntelRealSense/realsense-ros/files/4758740/test_27.txt)
[test_28.txt](https://github.com/IntelRealSense/realsense-ros/files/4758741/test_28.txt)
[test_29.txt](https://github.com/IntelRealSense/realsense-ros/files/4758742/test_29.txt)

**Comment by doronhi** [comment]
I fixed a similar issue in version 2.2.14. Could you check it out?

**Comment by RealSenseSupport** [comment]
The issue has been addressed since 2.2.14 with librealsense built using native backend.
Closing this issue. Thank you.
